yarn.lock

```diff
@@ -2715,6 +2784,21 @@
   dependencies:
     "@octokit/openapi-types" "^7.2.0"
 
+"@opentripplanner/core-utils@^3.2.4":
+  version "3.2.4"
+  resolved "https://registry.yarnpkg.com/@opentripplanner/core-utils/-/core-utils-3.2.4.tgz#68366f1527fe5a4593a0efe59207c45c0f54c2d2"
+  integrity sha512-3MYdErBl1mpMNj8qkuvyA77OokTRqwzAqcJt5AgIz6aEgPqXfx3znLtoUr/oBw3/OSjMDJB2cCzXEC/iFmGmnQ==
+  dependencies:
+    "@mapbox/polyline" "^1.1.0"
+    "@opentripplanner/geocoder" "^1.0.2"
+    "@turf/along" "^6.0.1"
+    bowser "^2.7.0"
+    lodash.isequal "^4.5.0"
+    moment "^2.24.0"
+    moment-timezone "^0.5.27"
```

Comment:
When merging #285 into this, please make sure `moment-timezone` is out of the lock file!

Reply:
Yep, moment-timezone is out after updating the core-utils dependency (see https://github.com/opentripplanner/otp-ui/pull/281/commits/7735f8b5d76bee70fe23841ee6a9d01a34da5238)